libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -355,7 +355,16 @@ export function useWalletAPIServer({
           currencies: currencyList,
           onSuccess: (account: AccountLike, parentAccount: Account | undefined) => {
             tracking.requestAccountSuccess(manifest);
-            resolve(accountToWalletAPIAccount(account, parentAccount));
+            const walletApiAccount: WalletAPIAccount = accountToWalletAPIAccount(
+              account,
+              parentAccount,
+            );
```

Comment:
I was thinking more of just adding the `parentAccountId` and not full `parentAccount`
This also mean that we only need to modify `accountToWalletAPIAccount` as it already gets the `parentAccount`